README.md

```diff
@@ -39,6 +42,33 @@ FROM $base_image
 See [alphagov/frontend/Dockerfile](https://github.com/alphagov/frontend/blob/-/Dockerfile) for a full, real-world example.
 
 
+## Supported tags
+
+Our version maintenance policy is the same as [upstream](https://www.ruby-lang.org/en/downloads/branches/): we support the three most recent major.minor releases of Ruby. For example in June 2024 those are `3.3`, `3.2`, `3.1`.
```

Comment:
This doesn't seem to be accurate, we're currently only building 3.2 and 3.3 images: https://github.com/alphagov/govuk-ruby-images/blob/main/build-matrix.json

Reply:
Whoops, thanks! I'll go update that.